services/cloudaccounts/azure/azure.go

```diff
@@ -86,3 +103,43 @@ func (service *Service) Delete(id string) (*http.Response, error) {
 
 	return resp, nil
 }
+
+func (service *Service) UpdateName(body CloudAccountUpdateNameRequest) (*CloudAccountResponse, *http.Response, error) {
+	v := new(CloudAccountResponse)
+	resp, err := service.Client.NewRequestDo("PUT", fmt.Sprintf("%s/%s", cloudaccounts.RESTfulPathAzure, cloudaccounts.RESTfulServicePathAzureName), nil, body, v)
```

Comment:
Does it work? because in API documentation I see that the path for that request be like:
`PUT /v2/AzureCloudAccount/{id}/OperationMode`
The `id` param isn't part of the body. It should be between the `cloudaccounts.RESTfulPathAzure` and the `cloudaccounts.RESTfulServicePathAzureName`.
There is inconsistency between this implementation and the API documentation

Reply:
I mean AccountName not OperationMode